<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoStatusOptionsWidgetClass</class>
 <widget class="QWidget" name="AutoStatusOptionsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>106</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hltAwayEnabled">
     <item>
      <widget class="QCheckBox" name="chbAwayEnable">
       <property name="text">
        <string>Change status to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAwayStatus"/>
     </item>
     <item>
      <widget class="QLabel" name="lblAwayAfter">
       <property name="text">
        <string>after</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbAwayTime">
       <property name="suffix">
        <string> minutes</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1440</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblAwayInactive">
       <property name="text">
        <string>of inactivity with message:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspAwayEnable">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hltAwayMessage">
     <item>
      <spacer name="hspAwayText">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lneAwayText"/>
     </item>
     <item>
      <widget class="QToolButton" name="tlbAwayHelp">
       <property name="text">
        <string notr="true">?</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hltOfflineEnable">
     <item>
      <widget class="QCheckBox" name="chbOfflineEnable">
       <property name="text">
        <string>Automatically disconnect after</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbOfflineTime">
       <property name="suffix">
        <string> minutes</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1441</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblOfflineInactive">
       <property name="text">
        <string>of inactivity</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspOfflineEnable">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblShowRules">
       <property name="text">
        <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;show-all&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Show all rules for the automatic change of status...&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspShowRules">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
